src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+import Login from "./components/Login/Login";
+
+import "./App.css";
+
+function App() {
+  return (
+    <div>
```

Comment:
Can we use fragment here ?

Reply:
maybe, he would do something with this <div>.